app/controllers/users_controller.rb

```diff
@@ -6,9 +6,26 @@ class UsersController < ApplicationController
   before_action :authenticate_user!, except: %i(new create public user_photo)
   respond_to :html
 
+  # The publisher/aspect_dropdown partial requiers a @stream that responds to publisher().public()
```

Comment:
I'd prefer to change the publisher/aspect_dropdown partial rather than introducing a dummy class as a workaround.